engine/runtime-integration-tests/src/test/java/org/enso/interpreter/test/AutoscopedConstructorTest.java

```diff
@@ -28,14 +28,15 @@ public static void prepareCtx() {
     ctx = ContextUtils.createDefaultContext(out);
   }
 
-  @Before
+  @After
   public void resetOut() {
```

Comment:
Reset `@Before` is desirable to clean any debris from previous executions.

Reply:
It also meant that we weren't cleaning up tests after runs. Double-edged sword.

Reply:
You may want to do both. `@Before` and `@After` then.